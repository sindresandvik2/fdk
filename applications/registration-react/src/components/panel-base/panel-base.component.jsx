import React from 'react';
import PropTypes from 'prop-types';
import './panel-base.scss';

export const PanelBase = props => {
  const { children } = props;

  return <div className="panel-base">{children}</div>;
};

PanelBase.propTypes = {
  children: PropTypes.node.isRequired
};
