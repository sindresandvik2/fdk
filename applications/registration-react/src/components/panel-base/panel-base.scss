@import '../../../node_modules/designsystemet/fdk-designsystem-bootstrap4/scss/colors.scss';

.panel-base {
  background-color: $color-white;
  border-radius: 5px;
  padding: 2em;
  word-wrap: break-word;
}