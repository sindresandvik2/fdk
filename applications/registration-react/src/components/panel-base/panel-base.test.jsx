import React from 'react';
import { shallow } from 'enzyme';
import { PanelBase } from './panel-base.component';

test('should render PanelBase success correctly with props', () => {
  const wrapper = shallow(<PanelBase type="success">Panel base</PanelBase>);
  expect(wrapper).toMatchSnapshot();
});
