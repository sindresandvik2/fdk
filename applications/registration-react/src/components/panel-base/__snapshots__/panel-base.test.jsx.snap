// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PanelBase success correctly with props 1`] = `
<div
  className="panel-base"
>
  Panel base
</div>
`;
